aries_vcx/src/protocols/connection/inviter/states/requested.rs

```diff
@@ -26,10 +27,6 @@ impl TheirDidDoc for Requested {
 impl ThreadId for Requested {
     //TODO: This should land in the threadlike macro.
```

Comment:
perhaps not valid todo anymore

Reply:
Mother of all nitpicks, Patrik 😅 .

The whole state machine would get a pass through during the upcoming refactor (even if it had the type state pattern implemented already). 
It's not valid anymore, but if we start discussing invalid/obsolete/bad/inefficient/illegible stuff in the code this PR will never end.